tests/test_wave_01.py

```diff
@@ -11,9 +10,9 @@ def test_get_tasks_no_saved_tasks(client):
     # Assert
     assert response.status_code == 200
     assert response_body == []
+    assert Task.query.all() == []
```

Comment:
I think an assert like this can be a good assert! But it would make more sense if this was testing something like making sure a single task was deleted from the database. In this case, where you are not calling any endpoints that would add data, this assert is more verifying that the database is empty, which is something you set up yourself in your test set-up (or lack of one in this test's case).